server/src/plugins/tx-monitor.ts

```diff
@@ -0,0 +1,69 @@
+import fp from 'fastify-plugin';
+import { FastifyInstance } from 'fastify';
+import { IncomingMessage, ServerResponse, Server } from 'http';
+
+declare module 'fastify' {
+  export interface FastifyInstance<
+    HttpServer = Server,
+    HttpRequest = IncomingMessage,
+    HttpResponse = ServerResponse
+  > {
+    authenticate: any;
+  }
+}
+
+const cron = require('node-cron');
+const fetch = require('node-fetch');
```

Comment:
This `require` worked for me using `import` without any problem and we should move them alongside the other imports.

```typescript
import cron from 'node-cron';
import fetch from 'node-fetch';
```